docs/91_version_history.rst

```diff
@@ -35,229 +35,186 @@ Release Date: 2023-10-17
-       - A problem with converting very long integers from JSON has been resolved. This would have manifested itself as a crash when handling datapoints that contain 64 bit integers above a certain value.
        - A number of issues regarding the usage of memory have been resolved, including some small memory leaks. The overall memory footprint of north services should also be reduced in some circumstances.
-       - The performance of the asset tracker system has been improved, resulting in an improvement in the ingress performance of the system.
-       - The Python south plugin mechanism has been updated to fix an issue with ingestion of nested data point values.
-       - A bug in the statistics rate API that would result in incorrect data being returned has been fixed.
-       - An issue that required the north service to be restarted if the source of data to send was changed in a running services has been resolved. Changing the data source no longer requires a restart of the north service.
-       - A bug that prevents notification rules being executed for readings with asset codes starting with numeric values has been resolved.
+       - An issue with the SQLite In Memory and the SQLite LB storage plugins that could result in incorrect data being stored has been resolved.
+       - An erroneous message was being produced when starting the system using the SQLite in memory storage plugin. This has now been resolved.
+       - Support has been improved for switching between different storage plugins that allows for correct schema creation when using different sqlite plugin variants for configuration and readings storage.
+       - An update has been done to the default SQLite storage plugin to enable it to handle a large number of distinct asset codes in the readings. Previously the plugin was limited in the number of assets it could support. When the number of asset codes gets large the performance of the plugin will be reduced slightly, however it will continue to ingest data.
+       - An issue that required the north service to be restarted if the source of data to send was changed in a running service has been resolved. Changing the data source no longer requires a restart of the north service.
+       - An issue that could sometimes cause a running north service to fail if the configuration for that service is updated has been resolved.
+       - A problem that prevents an updated service from restarting after an upgrade if HTTPS is used for the interface between services has been resolved.
        - A problem with incorrect timestamps in the readings graphs when the timezone of the machine was something other than UTC has been resolved.
+       - A number of inconsistencies in the timezone returned by API calls have been resolved. All API calls should return timestamps in the UTC timezone unless a timezones explicitly included in the response.
```

Comment:
I think the end of the sentence should read "...in the UTC timezone unless a _timezone is_ explicitly included in the response."